src/fetch_package.js

```diff
@@ -17,74 +17,30 @@ const vm = require("vm");
 const request = require('superagent');
 const logging = require("winston");
 
-const cache = require("./cache.js");
 const profile = require("./profile.js");
 const graphiteUtil = require("./graphite_util.js");
 
-// fetchPackage takes a cacheBehavior property, which is one of these:
-//    'yes': try to retrieve the object from the cache
-//    'no': do not try to retrieve the object from the cache (but
-//          still store it in the cache after retrieving it).
-//    'ignore': do not try to retrieve the object from the cache,
-//          nor try to store it in the cache.
-//    'ims': retrieve the object from the cache, but use the
-//          last-modified date on the result in order to do an
-//          if-modified-since query.
-// This variable controls the cache behavior that is used if the
-// user does not pass in a value for cacheBehavior for fetchPackage().
-let defaultCacheBehavior;
-
 // How many times we retry on 5xx error or similar, before giving up.
-let numRetries;
+const numRetries = 2;
 
 // What requests are currently in flight?
-let inFlightRequests;
-
-
-const resetGlobals = function() {
-    defaultCacheBehavior = 'yes';
-    numRetries = 2;     // so 3 tries total
```

Comment:
you may want to move the comment here to line 24

Reply:
Good catch, thanks!